src/controllers/postsController.js

```diff
@@ -62,7 +63,10 @@ export const sendPost = async (req, res) => {
     );
     if (content) {
       const tags = extractTags(content);
-      await processTags(tags, createdPost.id);
```

Comment:
This change can be in a single line `if (tags) await processTags(tags, createdPost.id);`